src/sidebar/services/view-filter.js

```diff
@@ -20,7 +20,7 @@ function displayName(ann) {
  * which do not match the filter are then hidden.
  */
 // @ngInject
-export default function viewFilter(unicode) {
+export default function ViewFilter(unicode) {
```

Comment:
Presumably because this was causing a false-negative with `isClass` in the new Injector...

Reply:
You're absolutely correct.